src/test/javascript/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "scripts": {
     "test": "node jasmine",
-    "integration-tests": "npm test -- integration/**/*_spec.js"
+    "integration-tests": "npm test -- integration/**/*_spec.js",
+    "integration-test": "npm test -- integration/**/$FILES"
```

Comment:
Is `$FILES` populated automatically by npm based on arguments or something? I've not seen that before.

Reply:
Ahh, ok. That makes sense.